python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -3161,6 +3163,103 @@ def add_backfill(self, partition_backfill: "PartitionBackfill") -> None:
     def update_backfill(self, partition_backfill: "PartitionBackfill") -> None:
         self._run_storage.update_backfill(partition_backfill)
 
+    @public
+    def launch_backfill(
+        self,
+        asset_graph: "BaseAssetGraph",
```

Comment:
I decided to directly ask for the asset graph as an argument, since there are already tutorials that ask the user to pass asset graph. I didn't really see any reason to ask for a broader `Definitions` object.